traits/category.py

```diff
@@ -64,28 +78,6 @@ def __new__ ( cls, class_name, bases, class_dict ):
         # Finish building the class using the updated class dictionary:
         return type.__new__( cls, class_name, bases, class_dict )
 
-#-------------------------------------------------------------------------------
-#  'MetaCategoryObject' class:
-#-------------------------------------------------------------------------------
-
-class MetaCategoryObject ( MetaHasTraitsObject ):
```

Comment:
There are still references to `MetaCategoryObject` and `MetaHasTraitsObject` in the documentation (`/docs/source/traits_api_reference/category.rst`).